main.cpp

```diff
@@ -356,6 +403,30 @@ int main(int argc, const char *argv[])
     // parse tweets
     auto tweets = chunks | parsetweets(poolthread, tweetthread);
 
+    /* Take tweets, make sure that any error is ignored, and publish them
+       as a stream of observables available to multiple on-demand subscribers.
+
+       If an error is encountered, function on_error_resume_next ensures the error
+       item is replaced with some correct observable value. In our case we return
+       an empty Tweet.
+       Publish creates a connectable observable that can be subscribed to later and that
+       starts operating when a consumer requests it
+       ref_count provides interface to connectable observable to consumers that
+       take ordinary observable. It also keeps the track, how many consumers are
+       connected to the observable, hence the name.
+       Repeat repeats the input given number of times. In our case we repeat it 0
+       times meaning that we return an empty sequence (?)
+
+       OnErrorResumeNext:
+         https://github.com/ReactiveX/RxJava/wiki/Error-Handling-Operators
+       Publish:
+         http://reactivex.io/documentation/operators/publish.html
+       RefCount:
+         http://reactivex.io/documentation/operators/refcount.html
+       Detailed explanation on Publish and RefCount (C#):
+         http://www.introtorx.com/content/v1.0.10621.0/14_HotAndColdObservables.html
+     */
+    // TODO: More explanation on repeat(0) is needed
```

Comment:
That seems inconsistent with some other Rx implementation (at least with RxJava 1.0, https://github.com/ReactiveX/RxJava/blob/1.x/src/main/java/rx/Observable.java#L8536 ). Should I file an issue to RxCpp repo?